Gordon360/Models/CCT/Housing/DueDate.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[Table("DueDate", Schema = "Housing")]
+public partial class DueDate
```

Comment:
This should not be its own table to store just a single value. I would recommend using the `[dbo].[Config]` table with a `[Key]` of `"Housing_Lottery_Due_Date"`

Reply:
Okay! Will do it!